src/Plugin/resource/ResourceEntity.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\restful\Plugin\resource\ResourceEntity.
+ */
+
+namespace Drupal\restful\Plugin\resource;
+
+use Drupal\restful\Exception\InternalServerErrorException;
+use Drupal\restful\Plugin\resource\DataProvider\DataProviderEntity;
+use Drupal\restful\Plugin\resource\Field\ResourceFieldCollectionInterface;
+
+class ResourceEntity extends Resource {
+
+  /**
+   * The entity type.
+   *
+   * @var string
+   */
+  protected $entityType;
+
+  /**
+   * The entity bundles.
+   *
+   * @var array
+   */
+  protected $bundles;
```

Comment:
Yeah, I could not remember the reason behind a single bundle. I figured that it would be easier to move from single to multiple than the other way around.